libindy/tests/anoncreds.rs

```diff
@@ -3539,14 +3540,13 @@ mod medium_cases {
                "version":"0.1",
                "requested_attributes": json!({}),
                "requested_predicates": json!({
-                   "predicate1_referent": json!({ "name":"age", "p_type":"<=", "p_value":18 }),
+                   "predicate1_referent": json!({ "name":"age", "p_type":"!=", "p_value":18 }),
```

Comment:
Where is the change in p_type described in the API documentation? Can you add it, and additional tests that would support the new predicate types that are being added?

Reply:
The change in p_types are not necessary at this point because the additional p_types aren't exposed at the API layer yet. This change is fine because it's a test to make sure the invalid p_type is properly throwing an error.